cpmonitor/templates/taskgroup.html

```diff
@@ -25,6 +25,11 @@
   {% endif %}
 {% endblock breadcrumbs %}
 {% block content %}
+  {% if city.assessment_status %}
```

Comment:
This will also be shown for a sector. I think, it would be better to do something like `if not node and city.assessment_status`.

Reply:
I will do that.